### PR TITLE
Introduced continue conversation section.

### PR DESCRIPTION
**Pull request by joshJarr**
continue-convo-section

### CODE REVIEW

File: src/assets/img/arrow-right-dark.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg height="14" viewBox="0 0 15 14" width="15" xmlns="http://www.w3.org/2000/svg"><path d="m6.0890411.95570776.70958904-.70958904c.30045662-.30045662.78630137-.30045662 1.08356164 0l6.21369862 6.21050228c.3004566.30045663.3004566.78630137 0 1.08356165l-6.21369862 6.21369865c-.30045662.3004566-.78630137.3004566-1.08356164 0l-.70958904-.7095891c-.30365297-.3036529-.29726028-.7990867.01278538-1.096347l3.85159818-3.66940639h-9.18630137c-.42511416 0-.76712329-.34200913-.76712329-.76712328v-1.02283106c0-.42511415.34200913-.76712328.76712329-.76712328h9.18630137l-3.85159818-3.6694064c-.31324201-.29726027-.3196347-.79269406-.01278538-1.09634703z" fill="#000"/></svg>
```
Comment (matthewhall) [inline src/assets/img/arrow-right-dark.svg]: Do we need this? We should be able to adjust the arrow's colour by setting its `fill` property in CSS.

File: src/index.html [hunk 2]
```diff
@@ -215,7 +215,37 @@ <h3>Thank you!</h3>
 
       <section class="section section--continue-conversation" id="section-continue-conversation">
         <div class="wrapper">
-          <p class="text-align-center">Continue the conversation.</p>
+          <div class="continue-conversation">
+            <div class="continue-conversation__intro">
+              <h2 class="h3">Continue the conversation</h2>
+              <h2 class="h2">#mdnWebDna</h2>
```
Comment (matthewhall) [inline src/index.html]: `h3`?

File: src/styles/components/_button.scss [hunk 3]
```diff
@@ -45,6 +45,10 @@
     transition: transform .2s;
   }
 
+  &--dark {
```
Comment (matthewhall) [inline src/styles/components/_button.scss]: If we change the `border-bottom` colour on the main `cta-link` declaration to `currentColor` I think we can lose this whole class.

File: src/styles/pages/_home.scss [hunk 4]
```diff
@@ -57,3 +57,77 @@
...
 }
+
+.continue-conversation {
+  @include mq($from: tablet) {
+    display: grid;
+    grid-gap: 16px;
+    grid-template-columns: repeat(8, 1fr);
+    grid-template-rows: repeat(6, 100px);
+  }
+
```
Comment (matthewhall) [inline src/styles/pages/_home.scss]: Nitpick: can we remove this line?

File: src/assets/img/arrow-right-dark.svg (hunk 1 above)
Comment (joshJarr) [reply src/assets/img/arrow-right-dark.svg]: Loading this as an image :( 
Though could probably load it as an inline SVG!

Comment (joshJarr) [reply src/assets/img/arrow-right-dark.svg]: Done that 👍

File: src/styles/components/_button.scss (hunk 3 above)
Comment (joshJarr) [reply src/styles/components/_button.scss]: 👍 Amazing stuff!

File: src/styles/pages/_home.scss (hunk 4 above)
Comment (joshJarr) [reply src/styles/pages/_home.scss]: 👍

**Comment by joshJarr** [comment]
@matthewhall All addressed, conflicts blocked, let me know if there's any more feedback!

**Pull request merged**